frontend/src/components/Navbar.js

```diff
@@ -87,7 +87,7 @@ const Navbar = () => {
                         }}
                         onClick={() => navigate("/createRoom")}
                     >
-                        Create Room
+                        Custom Room
```

Comment:
"Create Custom Room" could make more sense?

Reply:
kind of not sure about this because it looks too lengthy for a navigation button. Furthermore, its not only creatint but people join rooms through it as well.

Reply:
Ok, makes sense!